### DISCUSSION TITLE
How to build a theme demo with WP Playground blueprints

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#196] [Topic Ideas]
One of the most painful experiences (at least in my opinion) is managing live demos to showcase your new Wordpress theme to the world. You usually need to set up and maintain a multisite instance for multiple themes. And, it becomes more complex if you want to give potential users access to the admin to actually explore it. It's a headache and management nightmare.

What if you didn't need to do all of that?

That's where WordPress Playground comes in, especially its Blueprints API: https://wordpress.github.io/wordpress-playground/blueprints-api/index

You can easily provide a demo (even multiple demos with their own configurations) by simply providing a blueprint to Playground.

My proposal is to write a guide on using Playground to showcase your theme.

We have some working examples in the current Block Theme Examples repo (click the links in the README to see how this works): https://github.com/WordPress/block-theme-examples

**Comment by ironnysh** [comment]
That's a great idea!
I recently prepared something similar as part of a WP website handover. I used it to enrich the written documentation (also included the custom blocks as already-activated plugins). I'd be happy to brainstorm :-)

**Reply by justintadlock** [reply]
I'm happy to hear any ideas or see any examples. Honestly, I don't even know how to do this yet entirely on my own, so this is going to be a learning experience for me.

**Reply by ironnysh** [reply]
Great. I can prepare a Google Doc with an example (unfortunately can't share the full project), and include links to resources and some comments on the how and why. Would that work?

**Reply by justintadlock** [reply]
That'd be awesome!

**Comment by bph** [comment]
Discussion locked, please continue conversation on the issue

- #198

**Comment by bph** [comment]
The new issue, covers the topic of Theme demos via Playground. And still needs a writer. 

- #244
